inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java

```diff
@@ -66,6 +66,8 @@ public class InlongConstants {
 
     public static final String PULSAR_QUEUE_TYPE_PARALLEL = "PARALLEL";
 
+    public static final String PULSAR_CSV_FIELDDELIMITER = "fieldDelimiter";
```

Comment:
Suggested changing to `FIELD_DELIMITER`.

Reply:
done